src/front/glsl/builtins.rs

```diff
@@ -445,12 +445,19 @@ pub fn inject_builtin(declaration: &mut FunctionDeclaration, module: &mut Module
             // bits layout
             // bit 0 - dim part 1 - 1D/2D
             // bit 1 - array
-            // bit 2 - dim part 2 - 3D
+            // bit 2 - multisample
+            // bit 3 - dim part 2 - 3D
             //
-            // 0b100 is the latest since 3D arrayed images aren't allowed
-            for bits in 0..(0b101) {
-                let dim = bits & 0b1 | (bits & 0b100) >> 1;
+            // 0b1000 is the latest since 3D arrayed images aren't allowed
+            for bits in 0..(0b1001) {
+                let dim = bits & 0b1 | (bits & 0b1000) >> 2;
                 let arrayed = bits & 0b10 == 0b10;
+                let multi = bits & 0b100 == 0b100;
```

Comment:
it's easier to just write `bits & something != 0`

Reply:
it's was just keeping the same style as the other bit checks